NEWS.md

```diff
@@ -1,3 +1,8 @@
+## 2.0.0 IN PROGRESS
+
+* Upgrades to RAML-Module-Builder 33.0.0 (MODUSERBL-119)
+* Upgrades to Vert.x 4.1.0.CR1 (MODUSERBL-119)
```

Comment:
These news entries should refer to MODCXINV-56 instead of MODUSERSBL-119.

Also please include a line (which I forgot to do) about the removal of the parameter:
`embed_postgres command line option is no longer supported (MODCXINV-56)`


```suggestion
* embed_postgres command line option is no longer supported (MODCXINV-56)
* Upgrades to RAML-Module-Builder 33.0.0 (MODCXINV-56)
* Upgrades to Vert.x 4.1.0.CR1 (MODCXINV-56)
```